src/main/java/stringCalculator/StringCalculator.java

```diff
@@ -0,0 +1,75 @@
+package stringCalculator;
+
+import java.util.StringTokenizer;
+
+import static java.lang.Double.parseDouble;
+
+public class StringCalculator {
+
+    public double calculate(String text) {
+        if(text.equals(" ") || text.isEmpty()) throw new IllegalArgumentException("수식이 비었거나 null 입니다.");
```

Comment:
```suggestion
        if(text.equals(" ") || text.isEmpty()) {
                throw new IllegalArgumentException("수식이 비었거나 null 입니다.");
        }
```

작성해주신 스타일이 승재님의 컨벤션일까요!?
미션을 진행하는 동안은 자바 컨벤션에 맞게 구현하는 것이 좋을 것 같은데 어떻게 생각하시나요?

자바 컨벤션에 관련된 자료는 https://google.github.io/styleguide/javaguide.html 또는 https://myeonguni.tistory.com/1596 를 참고해주세요 :)